README.md

```diff
@@ -1,93 +1,16 @@
-
-Contributing directly to the docs works differently across the doc sets in this repo.
-
-### Observability Guide
-
-The source files for the Observability Guide are written in [AsciiDoc](https://docs.asciidoctor.org/asciidoc/latest/) and are built using [elastic/docs](https://github.com/elastic/docs).
-
-To build the docs locally:
-
-1. Check out the `elastic/docs` repository, along with any repositories that contain source files.
-2. Run the `build_docs` script, passing in the path to the `index.asciidoc` and resource paths to other repos that contain source files. For example, to build the Observability Guide and open it in the browser, run:
-    ```
-    ../docs/build_docs --doc ./docs/en/observability/index.asciidoc --chunk 3 --resource ../apm-server --resource ../ingest-docs/docs --open
-    ```
-
-The above command assumes that this repo, [elastic/docs](https://github.com/elastic/docs), [elastic/ingest-docs](https://github.com/elastic/ingest-docs), and [elastic/apm-server](https://github.com/elastic/apm-server) are checked out into the same parent directory.
-
-If you prefer to use aliases, you can load the [elastic/docs/doc_build_aliases.sh file](https://github.com/elastic/docs/blob/master/doc_build_aliases.sh), which has the resources defined for you.
-
-### Elastic Observability Serverless docs
-
-The source files for the Serverless docs are written in [AsciiDoc](https://docs.asciidoctor.org/asciidoc/latest/) and are built using [elastic/docs](https://github.com/elastic/docs).
-
-To build the docs locally:
-
-1. Check out the `elastic/docs` repository, along with any repositories that contain source files.
-2. Run the `build_docs` script, passing in the path to the `index.asciidoc` and resource paths to other repos that contain source files. For example, to build the Observability Guide and open it in the browser, run:
-    ```
-    ../docs/build_docs --doc ../docs-content/serverless/index.asciidoc --chunk 5 --open --resource ./docs/en/serverless --resource ../security-docs/docs/serverless --resource ../docs-content/serverless
-    ```
+> [!IMPORTANT]
+> Starting with Elastic Stack v9.0.0 the content in this repo has been moved to other repos.
+>
+> **If you need to update the docs for version 8.17 or earlier**, you can open a PR in this repo targeting the specific version branch and backport to other branches using the [Backport CLI tool](https://github.com/sorenlouv/backport?tab=readme-ov-file#backport-cli-tool).
+>
+> **If you're updating the docs for version 9.0 or later**, you can open a PR in the content's new home:
+>
+> * Observability guide and serverless docs: [elastic/docs-content](https://github.com/elastic/docs-content/tree/main/solutions/observability) (stateless and serverless docs have been consolidated into one doc set)
+> * Integrations developer guide: [elastic/integrations](https://github.com/elastic/integrations/tree/main/docs/extend)
```

Comment:
@bmorelli25 let me know what you think about this approach. After we've settled on the approach/language, I can open a similar PR in https://github.com/elastic/ingest-docs.

Reply:
One comment. This is really helpful, thank you